fetch.js

```diff
@@ -177,6 +177,8 @@
         this._bodyText = body.toString()
       } else if (!body) {
         this._bodyText = ''
+      } else if (body.getReader) {
+        this.body = body
```

Comment:
Yep, body is suppose to be a ReadableStream. You can test this directly in chrome's console log

```javascript
rs = new ReadableStream({})
res = new Response(rs)
reader = res.body.getReader()
console.log(reader)

// even this works:
res = new Response(new Blob([]))
reader = res.body.getReader()
console.log(reader)
```